pandas/tests/indexing/test_categorical.py

```diff
@@ -758,3 +758,23 @@ def test_map_with_dict_or_series(self):
         output = cur_index.map(mapper)
         # Order of categories in output can be different
         tm.assert_index_equal(expected, output)
+
+    def test_operations_with_interval_categories_index(self):
+        ind = pd.CategoricalIndex(pd.interval_range(start=0.0, end=2.0))
```

Comment:
Can you add the issue number as a comment?

Reply:
Will do